src/components/organisms/Card/Card.js

```diff
@@ -17,102 +17,65 @@ export default class Card extends HTMLElement {
 
-    // TODO: See CityOfDetroit/detroitmi#1099
-    // eslint-disable-next-line no-unused-vars
-    shadow.addEventListener('slotchange', (e) => {
-      // TODO: See CityOfDetroit/detroitmi#1099
-      // eslint-disable-next-line prefer-const
-      let tempElements = Array.from(this.children);
+    shadow.addEventListener('slotchange', () => {
+      const tempElements = Array.from(this.children);
       tempElements.forEach((node) => {
         switch (node.tagName) {
-          case 'COD-CARD-HEADER':
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line no-case-declarations, prefer-const
-            let tempHeader = document.createElement('div');
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line no-case-declarations, prefer-const
-            let tempHeaderClasses = ['card-header'];
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line eqeqeq
-            node.getAttribute('data-extra-classes') != undefined &&
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line eqeqeq
-            node.getAttribute('data-extra-classes') != null
+          case 'COD-CARD-HEADER': {
+            const tempHeader = document.createElement('div');
+            const tempHeaderClasses = ['card-header'];
+            node.getAttribute('data-extra-classes') !== null &&
+            node.getAttribute('data-extra-classes') !== null
               ? tempHeaderClasses.push(node.getAttribute('data-extra-classes'))
               : 0;
             tempHeader.className = tempHeaderClasses.join(' ');
             tempHeader.appendChild(node);
             this.card.appendChild(tempHeader);
             break;
-
-          case 'COD-CARD-BODY':
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line no-case-declarations, prefer-const
-            let tempBody = document.createElement('div');
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line no-case-declarations, prefer-const
-            let tempBodyClasses = ['card-body'];
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line eqeqeq
-            node.getAttribute('data-extra-classes') != undefined &&
-            // TODO: See CityOfDetroit/detroitmi#1099
-            // eslint-disable-next-line eqeqeq
-            node.getAttribute('data-extra-classes') != null
-              ? tempHeaderClasses.push(node.getAttribute('data-extra-classes'))
+          }
+          case 'COD-CARD-BODY': {
+            const tempBody = document.createElement('div');
+            const tempBodyClasses = ['card-body'];
+            node.getAttribute('data-extra-classes') !== null &&
+            node.getAttribute('data-extra-classes') !== null
+              ? tempBodyClasses.push(node.getAttribute('data-extra-classes'))
```

Comment:
This was a bug (copy-paste from the previous case statement). Previously used tempHeaderClasses.